WoWsShipBuilder.Web/Pages/Charts.razor

```diff
@@ -40,24 +34,27 @@
                 </MudText>
             }
         </MudButton>
-        <MudContainer>
-            <Chart Config="horizontalDispConfig"/>
-            <MudGrid Class="d-inline-flex" Style="flex-direction: column">
-                <MudItem Class="z-20 mt-1 ml-auto mr-3 d-inline-flex" xs="2" md="1" Style="position: absolute; align-self: flex-end">
-                    <MudSelect Label="@Localizer.GetAppLocalization(nameof(Translation.DispersionGraphWindow_PlotPlane)).Localization" @bind-Value="@selectedVertDispPlane" Variant="Variant.Outlined" SelectedValuesChanged="@(_ => ChangeVerticalDispPlane())" T="ChartsHelper.EllipsePlanes" Disabled="processing" Margin="Margin.Dense" AnchorOrigin="Origin.BottomRight">
-                        @foreach (var item in ellipsePlanesList)
-                        {
-                            <MudSelectItem Value="@item">
-                                <MudText Color="Color.Secondary">@Localizer.GetAppLocalization(Translation.ResourceManager.GetString(item.ToString()!)).Localization</MudText>
-                            </MudSelectItem>
-                        }
-                    </MudSelect>
-                </MudItem>
-                <MudItem Class="z-10" xs="12">
-                    <Chart Config="verticalDispConfig"/>
-                </MudItem>
-            </MudGrid>
-        </MudContainer>
+        <div>
+            <canvas id="@HorizontalDispersionId"></canvas>
```

Comment:
Is it possible to use @ref instead?

Reply:
No, chartjs specifically need a canvas with a set id. that is a variable simply so it's all centered in one location, without having to write it in every place,leading to possible typos. Ref is also not the same as id, since ref is an actual referece to the object, id is just an html tag.